tests/integration/targets/aci_criterion_vmattribute/tasks/main.yml

```diff
@@ -0,0 +1,196 @@
+# Test code for the ACI modules
```

Comment:
This module should be renamed and adjusted to manage all attribute types. Also new test should be defined for this.

Reply:
test module renamed, test case for type 'vmname' added